{
  "comments": [
    {
      "key": {
        "uuid": "a7c23755_794311c7",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 13,
      "author": {
        "id": 315
      },
      "writtenOn": "2016-12-21T10:13:06Z",
      "side": 1,
      "message": "This is the wrong bug number",
      "range": {
        "startLine": 13,
        "startChar": 5,
        "endLine": 13,
        "endChar": 11
      },
      "revId": "68125ed14b6a2d809546a0c0a2e6f6fda8cd0eaf",
      "serverId": "97ee7c02-f12f-4043-b43e-dea463d88b31",
      "unresolved": false
    }
  ]
}